src/main/java/com/smartbear/plugins/swaggerhub/ImportFromHubDialog.java

```diff
@@ -47,17 +48,22 @@
 import org.apache.http.entity.StringEntity;
 import org.json.JSONObject;
 
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 import static com.smartbear.swagger.AbstractSwaggerImporter.AUTHORIZATION_HEADER;
+import static javafx.scene.control.Alert.AlertType.WARNING;
```

Comment:
Use this for all types or import javafx.scene.control.Alert.AlertType only.

Reply:
Fixed